### PR TITLE
PEP 604: Add or operator to class type

### PR DESCRIPTION
**Pull request by mano7onam**
Python 3.10 allowing writing union types as X | Y so class ```type``` now should have operator __or__

### CODE REVIEW

File: stdlib/builtins.pyi [hunk 1]
```diff
@@ -156,6 +156,7 @@ class type(object):
     def __subclasscheck__(self, subclass: type) -> bool: ...
     @classmethod
     def __prepare__(metacls, __name: str, __bases: Tuple[type, ...], **kwds: Any) -> Mapping[str, Any]: ...
+    def __or__(self, t: type) -> type: ...
```
Comment (Akuli) [inline stdlib/builtins.pyi:159]: You could wrap this inside `if sys.version_info >= (3, 10)`

**Review by Akuli — COMMENTED** [review]
Why was this PR closed?

File: stdlib/builtins.pyi (hunk 1 above)
Comment (mano7onam) [reply stdlib/builtins.pyi:159]: Yes, I forgot about it. I'll do it and create new one

Comment (Akuli) [reply stdlib/builtins.pyi:159]: You can also change a pull request by pushing more commits to your branch.

**Comment by Akuli** [comment]
Replaced by #5051

File: stdlib/builtins.pyi (hunk 1 above)
Comment (mano7onam) [reply stdlib/builtins.pyi:159]: Ok, next time I will do this way

**Pull request closed without merging**